config/project/entryTypes/mediaEmbed--0dcf57f2-a803-4b2e-8235-bc0870989f38.yaml

```diff
@@ -0,0 +1,55 @@
+color: null
+fieldLayouts:
+  db50601d-9c20-49e1-8013-55a3def9b1e7:
+    cardView:
+      - 'layoutElement:0b14f3a0-3dae-49be-ad71-7cca5748fc40'
+    tabs:
+      -
+        elementCondition: null
+        elements:
+          -
+            dateAdded: '2025-02-25T02:50:10+00:00'
+            elementCondition: null
+            fieldUid: fad3fe56-0b6b-46c4-a97b-b7fb98847f3c # Media Embed
+            handle: null
+            includeInCards: false
+            instructions: null
+            label: null
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 71b3de71-a53a-4d7b-ae09-dc509291012f
+            userCondition: null
+            warning: null
+            width: 100
+          -
+            dateAdded: '2025-02-25T02:57:17+00:00'
+            elementCondition: null
+            fieldUid: 840acb32-6694-40d0-9384-60a0220c1de0 # Label
+            handle: caption
+            includeInCards: true
+            instructions: null
+            label: Caption
```

Comment:
Shouldn’t _Caption_ be an instance of the `simpletext` field so that it allows italics and links, just like in the _Image_ entry type?